### PR TITLE
Recording OEIS sequences that are formalized in the library

### PR DESCRIPTION
**Pull request by EgbertRijke**
This is a small pull request starting a file in which we can record which OEIS sequences are formalized. I also thought of a nice challenge, that I will perhaps record as an issue: Formalize the OEIS sequence with the smallest index that isn't already formalized.

### CODE REVIEW

File: src/online-encyclopedia-of-integer-sequences/oeis.lagda.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+A000244 : ℕ → ℕ
+A000244 = exp-ℕ 3
+
+A000720 : ℕ → ℕ
+A000720 = prime-counting-ℕ
+
+A001477 : ℕ → ℕ
+A001477 = id
+```
```
Comment (fredrik-bakke) [inline src/online-encyclopedia-of-integer-sequences/oeis.lagda.md:51]: Please add a link to the OEIS

File: src/online-encyclopedia-of-integer-sequences.lagda.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Online encyclopedia of integer sequences
+
+```agda
+module online-encyclopedia-of-integer-sequences where
+
+open import online-encyclopedia-of-integer-sequences.oeis public
+```
```
Comment (fredrik-bakke) [inline src/online-encyclopedia-of-integer-sequences.lagda.md:7]: Should this really have its own namespace? I'm not sure I like that. What about placing it in `elementary-number-theory` for now, or placing `oeis.lagda.md` plainly in `src`?

**Review by fredrik-bakke — APPROVED** [review]
This is a cool addition, and I like the idea!
It's a bit underwhelming that currently A000002 is not formalized though, although I don't expect this to be formalized anytime soon.

**Comment by EgbertRijke** [comment]
> This is a cool addition, and I like the idea!
> It's a bit underwhelming that currently A000002 is not formalized though, although I don't expect this to be formalized anytime soon.

Formalizing it should be a nice exercise. It shouldn't be very hard.

File: src/online-encyclopedia-of-integer-sequences.lagda.md (hunk 2 above)
Comment (fredrik-bakke) [reply src/online-encyclopedia-of-integer-sequences.lagda.md:7]: Alright, I agree it's not best practice to shoehorn this into `elementary-number-theory`. Perhaps individual files can be created in `online-encyclopedia-of-integer-sequences` when defining the number sequence isn't a one-liner and the sequence doesn't fit in another namespace.

**Comment by fredrik-bakke** [comment]
> > This is a cool addition, and I like the idea!
> > It's a bit underwhelming that currently A000002 is not formalized though, although I don't expect this to be formalized anytime soon.
> 
> Formalizing it should be a nice and relaxing exercise. It shouldn't be very hard.

I guess so, for those that are interested. 😄

**Pull request merged**